File: HoneyStore.DataAccess/Entities/Comment.cs
using HoneyStore.DataAccess.Identity;
using HoneyStore.DataAccess.Interfaces;

namespace HoneyStore.DataAccess.Entities
{
    public class Comment : IIdentifier
    {
        public int Id { get; set; }
        
        public int Mark { get; set; }

        public string Content { get; set; }

        public DateTime CreatedOn { get; set; }

        public int ProductId { get; set; }

        public Product Product { get; set; }

        public int UserId { get; set; }

        public User User { get; set; }
    }
}

File: HoneyStoreUI/src/app/models/comment.ts
export class Comment {
    id: number;
    productId: number; 
    userId: number;
    firstName: string;
    lastName: string;
    email: string;
    createdOn: Date;
    mark: number;
    content: string;
    userName: string;

    constructor(userId: number = 1, productId:  number = 1, content: string = '', mark: number = 0) {
        this.id = 0;
        this.userId = userId;
        this.firstName = '';
        this.lastName = '';
        this.email = '';
        this.productId = productId;
        this.mark = mark;
        this.userName = '';
        this.content = content;
        this.createdOn = new Date(Date.now());
    }
}